commands/addvertical.js

```diff
@@ -1,5 +1,6 @@
 const fs = require('fs-extra');
 const path = require('path');
+const { parse, stringify } = require('comment-json');
```

Comment:
Note: 'comment-json' was added in theme v1.17, which means when we create the updated patch files for themes older than 1.16, the patch files will need to modify the correct package.json file to include the dependency

Reply:
Good catch. Hopefully this won't make it much more difficult to craft the pre-1.16 patch files.

Reply:
As @oshi97 pointed out, comment-json is actually in the theme all the way back to v1.8 (and possibly a little earlier), so we probably don't have to worry about adding the dependency. I was mistakenly looking at package.json rather than static/package.json